core/src/datayoga_core/block.py

```diff
@@ -28,14 +28,14 @@ class Block(metaclass=ABCMeta):
         properties Dict[str, Any]: Block properties
     """
 
-    def __init__(self, properties: Dict[str, Any] = {}):
+    def __init__(self, properties: Dict[str, Any] = None):
```

Comment:
Shouldn't it be Optional[Dict[str, Any]] then?

Reply:
✅